src/main/resources/migrations.xml

```diff
@@ -3655,4 +3655,42 @@
 			<column name="customFields" type="${long_text_type}" />
 		</addColumn>
 	</changeSet>
+
+	<changeSet id="drop-tags-fts-mv" author="VassilIordanov" dbms="postgresql">
+		<sql>
+			DROP MATERIALIZED VIEW mv_fts_tags;
+		</sql>
+	</changeSet>
+
+	<changeSet id="convert-tags-to-notes" author="VassilIordanov">
+		<sql>
+			IF NOT EXISTS (SELECT uuid FROM people WHERE name = 'ANET Importer' AND role = 0)
+				INSERT INTO PEOPLE (uuid, name, status, role, createdAt, updatedAt)
+			        VALUES (${uuid_function}, 'ANET Importer', 1, 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+
+			INSERT INTO notes (uuid, "authorUuid", type, text, "createdAt", "updatedAt")
+				SELECT
+					tags.uuid,
+					(SELECT uuid FROM people WHERE name = 'ANET Importer' AND role = 0),
+					0,
+					CONCAT('Previously tagged as ', tags.name, ' - ',  tags.description),
+					tags."createdAt",
+					tags."updatedAt"
+				FROM
+					tags;
+
+			INSERT INTO "noteRelatedObjects" ("noteUuid", "relatedObjectType", "relatedObjectUuid")
+				SELECT
+					rt."tagUuid",
+					'reports',
+					rt."reportUuid"
+				FROM
+					"reportTags" rt;
+		</sql>
+		<dropForeignKeyConstraint baseTableName="reportTags" constraintName="fk_reportTags_report" />
+		<dropForeignKeyConstraint baseTableName="reportTags" constraintName="fk_reportTags_tag" />
+		<dropTable tableName="tags" />
+		<dropTable tableName="reportTags" />
```

Comment:
I would personally swap the order of these two statements, so first drop the m2m table `reportTags`, then drop `tags`.